### PR TITLE
Metrics app

### PR DESCRIPTION
**Pull request by aster-alemu**
In this PR, I have been doing the following activities.

1. Setup the environment
2. Choos the api
3. Create the components
4. test the components
5. check for linters errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,166 @@
...
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- [here]()
+ <!-- LIVE DEMO -->
+ ## 🚀 Live Demo <a name="live-demo"></a>
 
-## Learn More
+   [Live Demo Link](https://aster-alemu.github.io/metrics-web-app/)
```
Comment (t-yanick) [inline README.md]: - [ ] Great job working on this milestone. As per project requirements, you are supposed to deploy the application using Render, Netlify, or GitHub Pages. At the moment, your [Live demo link ](https://aster-alemu.github.io/metrics-web-app/) redirects to the GitHub repo of the project. Kindly review the link.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,166 @@
...
+- **Fetch Data from API**
+<!-- WALKTHROUGH -->
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 🤯 Walk through video <a name="walk-through"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+- [here]()
```
Comment (t-yanick) [inline README.md]: - [ ] Great job working on this milestone. As per project requirements, you are supposed to provide a link to your video presentation. At the moment, your [video presentation link ](https://aster-alemu.github.io/metrics-web-app/) redirects to the GitHub repo of the project. Kindly review the link.

File: src/components/homepage/HomepageList.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { getCompanies } from '../../redux/companySlice';
+import HomepageItem from './HomepageItem';
+import './HomepageList.css';
+
+const HomepageList = () => {
+  const navigate = useNavigate();
+  const [search, setSearch] = useState('');
+  const { companyList, status, error } = useSelector(
```
Comment (t-yanick) [inline src/components/homepage/HomepageList.js:12]: - [ ] Great job adding the test cases. At the moment, you have 1 test case failing. Kindly fix it.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @aster-alemu , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  The project contains a `Home` page and a `Detail` page.
:heavy_check_mark: The category item detail page has its unique route within the SPA
:heavy_check_mark: Use of correct Gitflow
:heavy_check_mark: App works locally as expected

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @aster-alemu, 

## Highlights!
- Good video presentation ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**